frontend/webapp/components/overview/add-entity/index.tsx

```diff
@@ -90,15 +90,15 @@ const AddEntity: React.FC<AddEntityButtonDropdownProps> = ({ options = DEFAULT_O
 
   return (
     <Container ref={dropdownRef}>
-      <StyledButton onClick={handleToggle}>
+      <StyledButton id='add-entity' onClick={handleToggle}>
```

Comment:
please use data-* (data-testid or a custom attribute like data-cy)

These attributes are specifically designed to be hooks for testing and are ignored by CSS and JavaScript functionality unless explicitly targeted

Reply:
Using id for testing might lead to conflicts with other functionality like CSS or JavaScript logic